AdyenCard/Components/Card/CardViewControllerItemsProvider.swift

```diff
@@ -34,7 +36,8 @@ extension CardViewController {
                       cardLogos: [FormCardLogosItem.CardTypeLogo],
                       scope: String,
                       defaultCountryCode: String,
-                      localizationParameters: LocalizationParameters?) {
+                      localizationParameters: LocalizationParameters?,
+                      addressViewModelBuilder: AddressViewModelBuilder) {
```

Comment:
card component should just use the default `AddressViewModelBuilder` without being injected

Reply:
Yes. I have updated the code.